### ISSUE TITLE
Not able to use the windows app version 

### ISSUE THREAD

**Issue by apatra9** [rams3sh/cowin4all#21]
I already had pip3 in my system, so I installed this package via pip3. It got installed and is importable in python3 shell, but the README says you have to run it via the terminal if you want the app version. When I try that from cmd or powershell, it says cowin4all is not recognized. How to make cowin4all a runnable command from terminal?

**Comment by rams3sh** [comment]
Hi @apatra9 

I didn't quiet understand your doubt.. 

It would be helpful if you could provide me a step by step command execution and it's corresponding output starting from installation and the final execution command, it can help me to solve the issue.

**Comment by apatra9** [comment]
Hi @rams3sh, I executed 2 commands on my terminal.

I don't have this output anymore, but I had executed 
`  pip3 install git+https://github.com/rams3sh/cowin4all.git  `

and the output was success one, where everything got installed. And then in the terminal I executed:

`cowin4all -e`
which gave output as:

`'cowin4all' is not recognized as an internal or external command,
operable program or batch file.`

Basically the readme needs to be updated to properly to include the installation of this package as an executable. Hope this helps.

**Comment by rams3sh** [comment]
Hey @apatra9 

The same commands were run by me below in my command prompt.  My logs of execution below. 

```cmd
C:\Users\ramac\Projects>python --version
Python 3.9.5

C:\Users\ramac\Projects>pip3 --version
pip 21.1.1 from c:\python39\lib\site-packages\pip (python 3.9)

C:\Users\ramac\Projects>git --version
git version 2.31.1.windows.1

C:\Users\ramac\Projects>pip3 uninstall cowin4all
WARNING: Skipping cowin4all as it is not installed.

C:\Users\ramac\Projects>cowin4all
'cowin4all' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\ramac\Projects>REM We do not have cowin4all as of now

C:\Users\ramac\Projects>pip3 uninstall cowin4all
WARNING: Skipping cowin4all as it is not installed.

C:\Users\ramac\Projects>REM Creating a virtual environment for ensuring clean environment
C:\Users\ramac\Projects>virtualenv env
created virtual environment CPython3.8.5.final.0-64 in 610ms
  creator CPython3Windows(dest=C:\Users\ramac\Projects\env, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=C:\Users\ramac\AppData\Local\pypa\virtualenv)
    added seed packages: attrs==21.2.0, certifi==2021.5.30, chardet==4.0.0, click==7.1.2, cowin4all==0.1, fastapi==0.65.1, h11==0.12.0, idna==2.10, jsonschema==3.2.0, pip==21.1.1, pydantic==1.8.2, pyrsistent==0.17.3, requests==2.25.1, setuptools==56.0.0, six==1.16.0, starlette==0.14.2, tabulate==0.8.9, typing_extensions==3.10.0.0, urllib3==1.26.5, uvicorn==0.13.4, wheel==0.36.2
  activators BashActivator,BatchActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
C:\Users\ramac\Projects>env\Scripts\activate
(env) C:\Users\ramac\Projects>pip3 install git+https://github.com/rams3sh/cowin4all.git
Collecting git+https://github.com/rams3sh/cowin4all.git
  Cloning https://github.com/rams3sh/cowin4all.git to c:\users\ramac\appdata\local\temp\pip-req-build-cn44n8tx
  Running command git clone -q https://github.com/rams3sh/cowin4all.git 'C:\Users\ramac\AppData\Local\Temp\pip-req-build-cn44n8tx'
Requirement already satisfied: requests==2.25.1 in c:\users\ramac\projects\env\lib\site-packages (from cowin4all==0.1) (2.25.1)
Requirement already satisfied: fastapi==0.65.1 in c:\users\ramac\projects\env\lib\site-packages (from cowin4all==0.1) (0.65.1)
Requirement already satisfied: uvicorn==0.13.4 in c:\users\ramac\projects\env\lib\site-packages (from cowin4all==0.1) (0.13.4)
Requirement already satisfied: tabulate~=0.8.9 in c:\users\ramac\projects\env\lib\site-packages (from cowin4all==0.1) (0.8.9)
Requirement already satisfied: jsonschema==3.2.0 in c:\users\ramac\projects\env\lib\site-packages (from cowin4all==0.1) (3.2.0)
Requirement already satisfied: setuptools~=56.0.0 in c:\users\ramac\projects\env\lib\site-packages (from cowin4all==0.1) (56.0.0)
Requirement already satisfied: pydantic!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0,>=1.6.2 in c:\users\ramac\projects\env\lib\site-packages (from fastapi==0.65.1->cowin4all==0.1) (1.8.2)
Requirement already satisfied: starlette==0.14.2 in c:\users\ramac\projects\env\lib\site-packages (from fastapi==0.65.1->cowin4all==0.1) (0.14.2)
Requirement already satisfied: attrs>=17.4.0 in c:\users\ramac\projects\env\lib\site-packages (from jsonschema==3.2.0->cowin4all==0.1) (21.2.0)
Requirement already satisfied: six>=1.11.0 in c:\users\ramac\projects\env\lib\site-packages (from jsonschema==3.2.0->cowin4all==0.1) (1.16.0)
Requirement already satisfied: pyrsistent>=0.14.0 in c:\users\ramac\projects\env\lib\site-packages (from jsonschema==3.2.0->cowin4all==0.1) (0.17.3)
Requirement already satisfied: chardet<5,>=3.0.2 in c:\users\ramac\projects\env\lib\site-packages (from requests==2.25.1->cowin4all==0.1) (4.0.0)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\ramac\projects\env\lib\site-packages (from requests==2.25.1->cowin4all==0.1) (2021.5.30)
Requirement already satisfied: idna<3,>=2.5 in c:\users\ramac\projects\env\lib\site-packages (from requests==2.25.1->cowin4all==0.1) (2.10)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\ramac\projects\env\lib\site-packages (from requests==2.25.1->cowin4all==0.1) (1.26.5)
Requirement already satisfied: click==7.* in c:\users\ramac\projects\env\lib\site-packages (from uvicorn==0.13.4->cowin4all==0.1) (7.1.2)
Requirement already satisfied: h11>=0.8 in c:\users\ramac\projects\env\lib\site-packages (from uvicorn==0.13.4->cowin4all==0.1) (0.12.0)
Requirement already satisfied: typing-extensions>=3.7.4.3 in c:\users\ramac\projects\env\lib\site-packages (from pydantic!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0,>=1.6.2->fastapi==0.65.1->cowin4all==0.1) (3.10.0.0)
WARNING: You are using pip version 21.1.1; however, version 21.1.2 is available.
You should consider upgrading via the 'C:\Users\ramac\Projects\env\Scripts\python.exe -m pip install --upgrade pip' command.

(env) C:\Users\ramac\Projects>cowin4all
Enter mobile number : 

Traceback (most recent call last):
  File "c:\users\ramac\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\users\ramac\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\ramac\Projects\env\Scripts\cowin4all.exe\__main__.py", line 7, in <module>
  File "c:\users\ramac\projects\env\lib\site-packages\cowin4all\app.py", line 268, in main
    confirmation = confirm_and_save_booking_details()
  File "c:\users\ramac\projects\env\lib\site-packages\cowin4all\app.py", line 172, in confirm_and_save_booking_details
    booking_details = get_booking_details(booking_details=booking_details)
  File "c:\users\ramac\projects\env\lib\site-packages\cowin4all\utils.py", line 507, in get_booking_details
    mobile_number = get_mobile_no()
  File "c:\users\ramac\projects\env\lib\site-packages\cowin4all\utils.py", line 288, in get_mobile_no
    mobile = input("Enter mobile number : ")
KeyboardInterrupt

(env) C:\Users\ramac\Projects>REM cowin4all command is working now. Lets see where it is located below.

(env) C:\Users\ramac\Projects\env>where cowin4all
C:\Users\ramac\Projects\env\Scripts\cowin4all.exe

(env) C:\Users\ramac\Projects\env>where python
C:\Users\ramac\Projects\env\Scripts\python.exe
C:\Python39\python.exe
C:\Users\ramac\AppData\Local\Programs\Python\Python38\python.exe
C:\Users\ramac\AppData\Local\Microsoft\WindowsApps\python.exe

(env) C:\Users\ramac\Projects\env>set | find /I "PATH"
ChocolateyLastPathUpdate=132670774389777404
HOMEPATH=\Users\ramac
Path=C:\Users\ramac\Projects\env\Scripts;C:\Python39\Scripts\;C:\Python39\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\ProgramData\chocolatey\bin;C:\Program Files\Git\cmd;C:\Users\ramac\AppData\Local\Programs\Python\Python38\Scripts\;C:\Users\ramac\AppData\Local\Programs\Python\Python38\;C:\Users\ramac\AppData\Local\Microsoft\WindowsApps;
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW
PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules
_OLD_VIRTUAL_PATH=C:\Python39\Scripts\;C:\Python39\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\ProgramData\chocolatey\bin;C:\Program Files\Git\cmd;C:\Users\ramac\AppData\Local\Programs\Python\Python38\Scripts\;C:\Users\ramac\AppData\Local\Programs\Python\Python38\;C:\Users\ramac\AppData\Local\Microsoft\WindowsApps;

```

I am able to get the cowin4all installed as per the README.

If you can replicate the commands exactly as mentioned above and provide me the output here similarly, I would be able to help.

**Comment by rams3sh** [comment]
Hey @apatra9 

Since there is no  response in this issue, I assume you were able to solve this.

Feel free to re-open this, incase issue still persists.
